.gitignore

```diff
@@ -1,2 +1,3 @@
 *.tgz
 .terraform/
+.idea/
```

Comment:
Better add this to your local `.gitignore` instad of maintaining it on all repos